specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,2239 @@
+      "enum": [
+        "CustomConversation"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeConversationTaskKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeConversationTaskResultsKind": {
+      "type": "string",
+      "description": "Enumeration of supported conversational task results",
+      "enum": [
+        "CustomConversationResult"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeConversationTaskResultsKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeConversationTaskResult": {
+      "description": "The base class of a conversation task result.",
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/AnalyzeConversationTaskResultsKind"
+        }
+      }
+    },
+    "CustomConversationalTask": {
+      "description": "The input for a custom conversation task.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationTask"
+        }
+      ],
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/ConversationAnalysisOptions"
+        },
+        "parameters": {
+          "$ref": "#/definitions/CustomConversationTaskParameters"
+        }
+      },
+      "x-ms-discriminator-value": "CustomConversation",
+      "required": [
+        "analysisInput",
+        "parameters"
+      ]
+    },
+    "CustomConversationTaskParameters": {
+      "type": "object",
+      "description": "Input parameters necessary for a CustomConversation task.",
+      "properties": {
+        "projectName": {
+          "type": "string",
+          "description": "The name of the project to use."
+        },
+        "deploymentName": {
+          "type": "string",
+          "description": "The name of the deployment to use."
+        },
+        "verbose": {
+          "type": "boolean",
+          "description": "If true, the service will return more detailed information in the response."
+        }
+      },
+      "required": [
+        "projectName",
+        "deploymentName"
+      ]
+    },
+    "CustomConversationalTaskResult": {
+      "description": "The results of a CustomConversation task.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationTaskResult"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "results": {
+          "$ref": "#/definitions/AnalyzeConversationResult"
+        }
+      },
+      "required": [
+        "results"
+      ],
+      "x-ms-discriminator-value": "CustomConversationResult"
+    },
+    "ConversationItemBase": {
+      "type": "object",
+      "discriminator": "modality",
+      "description": "The abstract base for a user input formatted conversation (e.g., Text, Transcript).",
+      "properties": {
+        "participantId": {
+          "description": "The participant Id of a ConversationItem",
```

Comment:
Nit: here and in other places, "ID" used in text should be all uppercase as a pseudo-acronym. Still use "Id" in the property name, though.

Reply:
updating the description.